projects/ui/src/Components/Apps/PageContent/AppsList.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from "react";
+import { di } from "react-magnetic-di";
+import { useListApps } from "../../../Apis/hooks";
+import { FilterPair, FilterType } from "../../../Utility/filter-utility";
+import { EmptyData } from "../../Common/EmptyData";
+import { Loading } from "../../Common/Loading";
+import { AppsPageStyles } from "../AppsPage.style";
+import { AppSummaryGridCard } from "./AppSummaryCards/AppSummaryGridCard";
+import { AppSummaryListCard } from "./AppSummaryCards/AppSummaryListCard";
+
+export function AppsList({
+  allFilters,
+  nameFilter,
+  usingGridView,
+}: {
+  allFilters: FilterPair[];
+  nameFilter: string;
+  usingGridView: boolean;
+}) {
+  di(useListApps);
+  const { isLoading, data: appsList } = useListApps(
+    "8e543407-cfdb-4061-b3ac-841faa715edf"
```

Comment:
Apps are listed per-team. The next PR will address listing all teams in the parent component, then combine the apps for all the teams together on this page.